src/pxPixels.h

```diff
@@ -53,7 +54,7 @@ struct pxPixel {
             unsigned char b: 8;
 #endif
         };
-        unsigned long u;
+        uint32_t u;

```

Comment:
unsigned long size is often 8 bytes on 64-but machine (I'm running 64-bit Linux Mint) 
http://stackoverflow.com/questions/7607502/sizeoflong-in-64-bit-c